src/sync/stream.ts

```diff
@@ -426,6 +427,12 @@ export class StreamManager extends AbortController {
       throw new Error('No abort controller on connection');
     }
 
+    // The connection with the fullnode may have been lost during the time waiting to sync
+    if (signal.aborted) {
+      this.abort();
+      throw new Error('The connection already aborted this stream');
```

Comment:
question(non-blocking): Does the scope of this PR allow adding a test for this abort condition?

Reply:
During the process to add this test I checked that this was unnecessary.
The idea was that the connection could be lost when waiting on the queue for execution, but the signal is created here.

I'll check how to create the abort controller before waiting on the queue, so if the connection is lost we can truly abort.